### ISSUE TITLE
reproduce validation accuracy

### ISSUE THREAD

**Issue by Hyeonjeong37** [Traffic-X/MonoLSS#2]
Hello,
Thank you for this great work and code!

I've tried to reproduce KITTI validation set accuracy. To this end, I've changed train_loader from 'trainval' to 'train' and kept the other conditions the same with original settings. As a result, I could obtain over 35% in Car category (3d moderate performance). It's much higher than the official performance in the paper, (24.77% in ablation studies).

Could you tell me the hyper-parameter settings or any differences when you train the model that shows the validation set performance?

**Comment by jjrCN** [comment]
Maybe it’s a problem with the number of max epoch. In order to compare fairly with other methods, we use 150 epochs for training on the train set.

**Comment by Hyeonjeong37** [comment]
Thank you for the reply!
I have one more question about this. If I train the model for 150 epochs, the performance is much less than the official performance in ablation studies. I got 20~21%, not 24%.. How could you set the learning rate decay?

**Comment by jjrCN** [comment]
you may try this setting:
decay_rate: 0.1
decay_list: [90, 120]

**Comment by Hyeonjeong37** [comment]
Thank you so much!!
